scripts/rollup/build.js

```diff
@@ -109,16 +109,19 @@ function getBabelConfig(updateBabelOptions, bundleType, filename) {
   }
 }
 
-function getRollupOutputOptions(outputPath, format, globals, globalName) {
+function getRollupOutputOptions(outputPath, format, globals, bundle) {
   return Object.assign(
     {},
     {
       file: outputPath,
       format,
       globals,
       interop: false,
-      name: globalName,
+      name: bundle.global,
       sourcemap: false,
+      amd: {
+        id: bundle.entry,
+      }
```

Comment:
needs trailing comma (just run `yarn prettier`)

Reply:
I thought I'd run that.  I'll double-check.

Reply:
Fixed.